build.gradle

```diff
@@ -0,0 +1,143 @@
+plugins {
+    id 'java'
+    id 'com.github.johnrengelman.shadow' version '2.0.1'
+    id "com.github.maiflai.scalatest" version "0.16"
+}
+
+group 'com.scality.clueso'
+version version
+
+apply plugin: 'java'
+apply plugin: 'scala'
+apply plugin: 'idea'
+apply plugin: 'com.github.johnrengelman.shadow'
+apply from: 'gradle/docker.gradle'
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+configurations {
+    provided
+    compile.extendsFrom provided
+}
+
+sourceSets {
+    main {
+        compileClasspath += configurations.provided
+
+        resources {
+            filter.exclude("application.conf")
+        }
+    }
+}
+
+repositories {
+    jcenter()
+    mavenLocal()
+    mavenCentral()
+    maven {
+        url "https://oss.sonatype.org/content/repositories/snapshots"
+    }
+}
+
+dependencies {
+    shadow group: 'org.scala-lang', name: 'scala-library', version: '2.11.8'
+    shadow group: 'org.scala-lang', name: 'scala-reflect', version: '2.11.8'
+    shadow group: 'org.scala-lang', name: 'scala-compiler', version: '2.11.8'
```

Comment:
for my knowledge, why do you have to do all 3 separately?

Reply:
they are different dependencies